krun/platform.py

```diff
@@ -21,21 +21,49 @@ def __init__(self):
         # errors and warnings in the Linux dmesg. If that happens, we
         # really want to know about it!
         self.current_dmesg = self._collect_dmesg_lines()
+        self.current_dmesg_time = time.time()
```

Comment:
former: I'm using it to convert to local time, but I see a workaround. (time api in python is awful).

latter: as you wish